lib/mastodon/cli_helper.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+dev_null = Logger.new('/dev/null')
+
+Rails.logger                 = dev_null
+ActiveRecord::Base.logger    = dev_null
+HttpLog.configuration.logger = dev_null
+Paperclip.options[:log]      = false
```

Comment:
Why disable logging entirely?

Reply:
Clean stdout, otherwise it's just a mess of ActiveRecord/Paperclip messages. This is ported over from the rake file!

Reply:
Hm, ok. This sounds like this could still be useful, though. Maybe make that configurable with a `--verbose` switch for instance?

Reply:
Thor doesn't seem to have a callback system so that would require very repetetive code in every method, right now it's just two but I still don't like that.